content/html/ibm-z-request-ci.html

```diff
@@ -0,0 +1,141 @@
+      <div class="form-item webform-component" id="webform-component-platforms">
+        <label for="webform-component-platforms">OS Platform(s) <span class="form-required"
+            title="This field is required.">*</span></label>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-centos" name="centos" class="form-checkbox" />
+          <label for="edit-submitted-centos">CentOS</label></div>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-fedora" name="fedora" class="form-checkbox" />
+          <label for="edit-submitted-fedora">Fedora</label></div>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-debian" name="debian" class="form-checkbox" />
+          <label for="edit-submitted-debian">Debian</label></div>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-ubuntu" name="ubuntu" class="form-checkbox" />
+          <label for="edit-submitted-ubuntu">Ubuntu</label></div>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-gentoo" name="gentoo" class="form-checkbox" />
+          <label for="edit-submitted-gentoo">Gentoo</label></div>
+        <div class="form-inline"><input type="checkbox" id="edit-submitted-other"  name="other" class="form-checkbox" />
+          <label for="edit-submitted-other">Other (please specify in the "Other Information" field above)</label></div>
+        <div class="description">Some platforms may not be supported on a specific architecture, so please check with
+            us first.</div>
+      </div>
+      <div class="form-item webform-component webform-component-textfield" id="webform-component-ci-github">
+        <label for="edit-submitted-ci-github">GitHub Username(s) for IBM Z CI <span class="form-required"
+            title="This field is required.">*</span></label>
+        <input type="text" id="edit-submitted-ci-github" name="ci-github" value="" size="60" maxlength="128"
+            class="form-text required" />
+        <div class="description">We use Github OAuth to authenticate into the CI system. Please provide a comma
+        separated list of Github username(s) to gain access.</div>
+      </div>
+
+      <p><i>You should receive an automated email from our request ticketing system to the email address you have
+      provided within 5-10 minutes.  If you don't receive this email please reach out to us at 
+      <!-- TODO: Same queue/alias or not? -->
+      <a href="mailto:powerdev-request@osuosl.org">powerdev-request@osuosl.org</a> or via IRC in <b>#osuosl</b> on
+      Freenode.</i></p>
+
+      <!-- Formsender Settings -->
+      <input type="hidden" name="last_name" value="" />
+      <input type="hidden" name="token" value="15674hsda//*q23%^13jnxccv3ds54sa4g4sa532323!OoRdsfISDIdks38*(dsfjk)aS" />
+      <!-- TODO: Is this still true? The following must be set to http://www.osuosl.org/services/powerdev/request_hosting in production -->
```

Comment:
@ramereth I noticed this comment on all of the forms, but not sure what the significance of it is, as we aren't using that destination in production.